examples-new/hosted-oauth/README.md

```diff
@@ -0,0 +1,84 @@
+# Example: Hosted OAuth
+
+This is an example project that demonstrates how to connect to Nylas via
+OAuth. [OAuth](https://oauth.net/) is a standard protocol to allow two
```

Comment:
Can you specify "via OAuth using Nylas' hosted oauth flow" or something similar, and link to our hosted oauth docs please? https://docs.nylas.com/reference#oauth